src/index.js

```diff
@@ -112,6 +141,20 @@ function activateTaskInputListeners() {
   });
 }
```

Comment:
The functions activateMoreListeners, activateCheckboxListeners, activateTaskInputListeners, and activateCompletedListener share similar patterns. Consider creating a generic function to reduce duplication.